### DISCUSSION TITLE
can I use 'where' param with condition of nested TableARead ? [Apr 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1983] [Discord Threads]
> 🙋‍♂️ **BoMBx**  
#### ```  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/968465614328954880/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **BoMBx**  
#### does it nocodb support for this ?  
#  
> 👨‍🚀 **wingkwong**  
#### Not supported at this moment  
#  
> 👨‍🚀 **wingkwong**  
#### however, a possible workaround is that u can create a lookup on it, then filter by lookup  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968465615176228904)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
